examples/features/profiling/README.md

```diff
@@ -0,0 +1,260 @@
+server/client distinction from the previous subsection), you need to enable
+profiling. There are three ways to do this -- at initialization, remotely
+post-initialization, or programmatically within Go.
+
+### Enabling Profiling at Initialization
+
+To force profiling to start measuring data right from the first RPC, set the
+`Enabled` attribute of the `ProfilingConfig` struct to `true` when you are
+initializing profiling.
+
+```go
+	// Set Enabled: true to turn profiling on at initialization time.
+	profsvc.Init(&profsvc.ProfilingConfig{
+		Server:  s,
+		Enabled: true,
+	})
+```
+
+### Enabling/Disabling Remotely
+
+Alternatively, you can enable/disable profiling any time after server
+initialization by using a bundled command-line tool designed for remote
+profiling management. Assuming `example.com:50051` is the address of the server
+that you would like to enable profiling in, do the following:
+
+```bash
+$ go run google.golang.org/grpc/profiling/cmd \
+    -address example.com:50051                \
+    -enable-profiling
+```
+
+Similarly, running the command with `-disable-profiling` can be used to disable
+profiling remotely.
+
+
+### Enabling/Disabling Within Go
+
+In addition to the remote service that is exposed, you may enable/disable
+profiling within your application in Go:
+
+```go
+import (
+	"google.golang.org/grpc/profiling"
+)
+
+func setProfiling(enable bool) {
+	profiling.Enable(true)
+}
+```
+
+The `profiling.Enable` function can be safely accessed and called concurrently.
+
+## Downloading and Processing Profiling Data
+
+Once your server has collected enough profiling data, you may want to download
+that data and perform some analysis on the retrieved data. The aforementioned
+command-line application within gRPC comes bundled with support for both
+operations.
+
+To retrieve profiling data from a remote server, run the following command:
+
+```bash
+$ go run google.golang.org/grpc/profiling/cmd \
+    -address example.com:50051                \
+    -retrieve-snapshot                        \
+    -snapshot /path/to/snapshot
+```
+
+You must provide a path to `-snapshot` that can be written to. This file will
+store the retrieved data in a raw and binary form.
+
+To process this data into a human-consumable such as
+[Catapult's trace-viewer format](https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview):
+
+```bash
+$ go run google.golang.org/grpc/profiling/cmd \
+    -address example.com:50051                \
+    -snapshot /path/to/snapshot               \
+    -stream-stats-catapult-json /path/to/json
+```
+
+This would read the data stored in `/path/to/snapshot` and process it to
+generate a JSON format that is understood by Chromium's
+[Catapult project](https://chromium.googlesource.com/catapult).
+The Catapult project comes with a utility called
+[trace-viewer](https://chromium.googlesource.com/catapult/+/HEAD/tracing/README.md),
```

Comment:
What's the easiest way to install/download catapult (or trace-viewer if it's separate)?
Do they have releases? Or just git clone?

Reply:
Yes. Added some additional instructions.